templates/1.10/bootstrap.conf.j2

```diff
@@ -0,0 +1,85 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Java command to use when running NiFi
+java={{ nifi_java_command }}
+
+# Username to use when running NiFi. This value will be ignored on Windows.
+run.as={{ nifi_user }}
+
+# Configure where NiFi's lib and conf directories live
+lib.dir={{ nifi_nar_dir }}
+conf.dir={{ nifi_conf_dir }}
+
+# How long to wait after telling NiFi to shutdown before explicitly killing the Process
+graceful.shutdown.seconds={{ nifi_graceful_shutdown_secs }}
+
+# Disable JSR 199 so that we can use JSP's without running a JDK
+java.arg.1=-Dorg.apache.jasper.compiler.disablejsr199=true
+
+# JVM memory settings
+java.arg.2=-Xms{{ nifi_node_jvm_memory }}
+java.arg.3=-Xmx{{ nifi_node_jvm_memory }}
+
+# Enable Remote Debugging
+{% if nifi_enable_remote_debugging -%}
+java.arg.debug=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address={{ nifi_remote_debugging_port }}
+{% else -%}
+#java.arg.debug=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address={{ nifi_remote_debugging_port }}
+{% endif -%}
+
+java.arg.4=-Djava.net.preferIPv4Stack=true
+
+# allowRestrictedHeaders is required for Cluster/Node communications to work properly
+java.arg.5=-Dsun.net.http.allowRestrictedHeaders=true
+java.arg.6=-Djava.protocol.handler.pkgs=sun.net.www.protocol
+
+# The G1GC is still considered experimental but has proven to be very advantageous in providing great
+# performance without significant "stop-the-world" delays.
+java.arg.13=-XX:+UseG1GC
+
+#Set headless mode by default
+java.arg.14=-Djava.awt.headless=true
+
+# Master key in hexadecimal format for encrypted sensitive configuration values
+nifi.bootstrap.sensitive.key=
+
+# Sets the provider of SecureRandom to /dev/urandom to prevent blocking on VMs
+java.arg.15=-Djava.security.egd=file:/dev/urandom
+
+# Requires JAAS to use only the provided JAAS configuration to authenticate a Subject, without using any "fallback" methods (such as prompting for username/password)
+# Please see https://docs.oracle.com/javase/8/docs/technotes/guides/security/jgss/single-signon.html, section "EXCEPTIONS TO THE MODEL"
+java.arg.16=-Djavax.security.auth.useSubjectCredsOnly=true
+
```

Comment:
Can you add the `nifi_tmp_dir` block from templates/1.9/bootstrap.conf.j2? My project needs that because of restrictions on the system /tmp dir:

`$ diff templates/1.9/bootstrap.conf.j2 templates/1.10/bootstrap.conf.j2
68,73d67
< {% if nifi_tmp_dir is defined %}
< # Specify java.io.tmpdir to workaround
< # http://apache-nifi-users-list.2361937.n4.nabble.com/nifi-hive-nar-1-7-1-nar-will-not-load-td5337.html
< java.arg.tmpdir=-Djava.io.tmpdir={{ nifi_tmp_dir }}
<
< {% endif %}
`